webpack/assets/javascripts/react_app/components/Bookmarks/components/BookmarkForm/index.js

```diff
@@ -1,15 +1,17 @@
 import { connect } from 'react-redux';
-
 import BookmarkForm from './BookmarkForm';
 import { submitForm } from '../../../../redux/actions/common/forms';
 
-export default connect(
-  ({ bookmarks }) => ({
-    initialValues: {
-      public: true,
-      query: bookmarks.currentQuery || '',
-      name: '',
-    },
-  }),
-  { submitForm }
-)(BookmarkForm);
+const mapStateToProps = ({ bookmarks }) => ({
+  initialValues: {
+    public: true,
+    query: bookmarks.currentQuery || '',
+    name: '',
+  },
+});
+
+const mapDispatchToProps = {
+  submitForm,
+};
```

Comment:
can you use the new convention with `useDispatch` and `useSelectors` ?

Reply:
@laviro It would work fine but I don't think it's in the scope of this PR, also my general rule is "build new things with Hooks, but don't bother refactoring just so you can use hooks"

Reply:
I agree, but sometimes when we touch old code its better to maintain it,
who knows when will be the next time someone will touch this area :)
I am ok with continuing this refactor in a follow up PR or adding a comment..

looking deeper on this file, it seem that we don't really need it either,
the only need here is to get the current searchQuery from the searchbar which is already passing down that query to bookmarks, see my other comments about it.